api_tests/metaschemas/views/test_metaschemas_detail.py

```diff
@@ -1,52 +1,60 @@
-from nose.tools import *  # flake8: noqa
-
-from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
-from website.project.model import Q
-from osf.models import MetaSchema
+import pytest
 
 from api.base.settings.defaults import API_BASE
-
-from tests.base import ApiTestCase
+from osf.models import MetaSchema
 from osf_tests.factories import (
     AuthUserFactory
 )
-class TestMetaSchemaDetail(ApiTestCase):
-    def setUp(self):
-        super(TestMetaSchemaDetail, self).setUp()
-        self.user = AuthUserFactory()
-        self.schema = MetaSchema.find_one(Q('name', 'eq', 'Prereg Challenge') & Q('schema_version', 'eq', LATEST_SCHEMA_VERSION))
-        self.url = '/{}metaschemas/{}/'.format(API_BASE, self.schema._id)
-
-    def test_pass_authenticated_user_can_retrieve_schema(self):
-        res = self.app.get(self.url, auth=self.user.auth)
-        assert_equal(res.status_code, 200)
+from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
+from website.project.model import Q
+
+@pytest.mark.django_db
+class TestMetaSchemaDetail:
```

Comment:
Technically since this class now consists of one test fixtures aren't really needed, but I'm not sure @alexschiller